.devcontainer/Dockerfile

```diff
@@ -1,3 +1,4 @@
+# This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json
```

Comment:
This is needed to make CI happy, fails also on main currently, same for license checks

Reply:
Yes with the new version of the cli (Which is part of the new base image) this dockerfile is maintained through our devcontainer-setup. We forgot to update the versions inside velocitas.json (Just for information, the approach is -> Starting the template in devcontainer and execute: velocitas upgrade --ignore-bounds && velocitas init && velocitas sync)

Reply:
Doesn't really matter for me, if you want to fix it in a separate PR I can later rebase on that PR.

Reply:
created a PR
https://github.com/eclipse-velocitas/vehicle-app-cpp-template/pull/106